### PR TITLE
upgrade-1x-to-2x/upgrade-2x: update progress messages based on feedback

### PR DESCRIPTION
**Pull request by imrehg**

### CODE REVIEW

File: upgrade-1.x-to-2.x.sh [hunk 1]
```diff
@@ -492,7 +492,7 @@ case $root_part in
                 ;;
         esac
         log "Rebooting..."
-        progress 15 "ResinOS: rebooting to continue..."
+        progress 15 "Rebooting from switched partition"
```
Comment (alisondavis17) [inline upgrade-1.x-to-2.x.sh]: maybe "second" or "new" would be clearer than "switched?" it's not clear which partition is meant by "switched"

File: upgrade-1.x-to-2.x.sh [hunk 2]
```diff
@@ -740,14 +740,14 @@ BACKUPARCHIVE=/tmp/backup/newos.tar.gz
...
-progress 50 "ResinOS: downloading OS update..."
+progress 50 "Downloading OS update"
 # Create container for new version
 CONTAINER=$(docker create ${IMAGE} echo export)
 if [ -z "$CONTAINER" ]; then
     log ERROR "Could not download target update image..."
 fi
 
-progress 60 "ResinOS: processing update package..."
+progress 60 "Processing OS update"
```
Comment (alisondavis17) [inline upgrade-1.x-to-2.x.sh]: maybe "processing OS update package"?

File: upgrade-1.x-to-2.x.sh (hunk 1 above)
Comment (imrehg) [reply upgrade-1.x-to-2.x.sh]: Neither (the pr or the alternatives) feel really good... will keep thinking.

File: upgrade-1.x-to-2.x.sh (hunk 2 above)
Comment (imrehg) [reply upgrade-1.x-to-2.x.sh]: Yeah, will update it. That will make it pretty much as long as the original was, wonder if there's any more concise way to express something similar.

File: upgrade-1.x-to-2.x.sh [hunk 3]
```diff
@@ -888,10 +888,10 @@ sync
...
     # Reboot into new OS
     log "Rebooting into new OS in 5 seconds..."
-    progress 100 "ResinOS: update successful, rebooting..."
+    progress 100 "Update successful, rebooting"
     nohup bash -c " /bin/sleep 5 ; /sbin/reboot " > /dev/null 2>&1 &
 else
     log "Finished update, not rebooting as requested."
     log "NOTE: Supervisor and stopped services kept stopped!"
-    progress 100 "ResinOS: update successful."
+    progress 100 "Update successful."
```
Comment (agherzan) [inline upgrade-1.x-to-2.x.sh]: I would remove the full stop from this too as you seem to have removed them from all the other messages.

Comment (imrehg) [reply upgrade-1.x-to-2.x.sh]: Yup, that's good.

**Comment by imrehg** [comment]
@alisondavis17 @agherzan updated based on suggestions, thanks a bunch!

**Pull request merged**